modules/main/ui/search/breadcrumb-navigation.tsx

```
@@ -0,0 +1,44 @@
+import {
+  Breadcrumb,
+  BreadcrumbEllipsis,
```

ESLint

[error] 'BreadcrumbEllipsis' is defined but never used.

(@typescript-eslint/no-unused-vars)